src/commands/refund_swap.rs

```diff
@@ -0,0 +1,50 @@
+use candid::Encode;
+use clap::Parser;
+use ic_sns_swap::pb::v1::ErrorRefundIcpRequest;
+
+use crate::{
+    lib::{
+        signing::{sign_ingress_with_request_status_query, IngressWithRequestId},
+        AnyhowResult, TargetCanister,
+    },
+    SnsCanisterIds,
+};
+
+use super::transfer;
+
+/// Signs a message to request a refund from the SNS swap canister.
+/// If the swap was aborted or failed, or some of your contributed ICP never made it into a neuron,
+/// this command can retrieve your unused ICP, minus transaction fees.
+#[derive(Parser)]
+pub struct RefundSwapOpts {
```

Comment:
Given what the description is, maybe a better name would be "GetSwapRefund"?  "RefundSwap" sounds like an action taken on the other side of the transaction